src/main/scala/SerialAdapter.scala

```diff
@@ -97,6 +97,25 @@ case object SerialAdapter {
     ram
   }
 
+  def connectControlOnlyHarness(desser: TLDesser, port: SerialIO, reset: Reset): SerialHarness = {
+    implicit val p: Parameters = desser.p
+
+    val harness = LazyModule(new SerialHarness(
+      p(SerialTLKey).get.width,
+      clientEdge = desser.node.edges.out(0)
+    ))
+
+    val module = Module(harness.module)
+    module.io.ser <> port
+
+    /*require(harness.serdes.module.mergeTypes.head.param == desser.module.mergeTypes.head.param,
```

Comment:
in `connectHarnessRAM` there is a require here that works, I tried to match the parameters to what they are in the `desser` and `serdes` but this seems fire even when the simulation as a whole works

Reply:
out of band: removing this require is fine.